### ISSUE TITLE
Stuck on "Fetching new git commits"

### ISSUE THREAD

**Issue by millman2394** [mina-deploy/mina#654]
After running `mina deploy` I get the following:

```
-----> Quiet sidekiq (stop accepting new work)
       /var/www/myproject/tmp/build-157262358827785
-----> Fetching new git commits
```

It stays here and nothing happens

Edit, when I press ctrl + c, it will say

```
-----> Mina: SIGINT received.
       Enter passphrase for key '/root/.ssh/id_rsa':
       Elapsed time: 1931.59 seconds
```

In my `deploy.rb` file I have `set :term_mode, nil`

There's no way to enter a password

**Comment by millman2394** [comment]
Okay so I found this closed github issue: https://github.com/mina-deploy/mina/issues/449

I found I simply needed to add `set :execution_mode, :system` to my deploy file.

The answer on the github issue says it's on the FAQ, however, their wording is the following:

```
- Mina hangs after i type my password in
Mina assumes that you have set up the communication with your server through the public/private keys, not password. If you want to use the password you will have to change the execution mode:
```

I was never even prompted for a password, so I didn't know this applied to me. I also AM using ssh public/private keys (obviously) so the way they worded this is strange. maybe it's because I'm not too experienced with this.

Anyway, in case someone has the same issue - here's the answer.
